config/initializers/swagger_blocks.rb

```diff
@@ -0,0 +1,34 @@
+module ApiFlashcards
+  include Swagger::Blocks
+    swagger_root do
+      key :swagger, '2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.